libs/ruby.json

```diff
@@ -33,5 +33,6 @@
   {"id":"rb.dry-equalizer","imports":["dry-equalizer"],"name":"dry-equalizer","repo":"dry-rb/dry-equalizer","tags":["compare"],"tech":["utils"],"status":"model-ready"},
   {"id":"rb.dry-inflector","imports":["dry/inflector"],"name":"dry-inflector","repo":"dry-rb/dry-inflector","tags":["inflector"],"tech":["utils"],"status":"model-ready"},
   {"id":"rb.dry-logic","imports":["dry/logic"],"name":"dry-logic","repo":"dry-rb/dry-logic","tags":["predicate-logic"],"tech":["utils"],"status":"model-ready"},
-  {"id":"rb.dry-types","imports":["dry-types"],"name":"dry-types","repo":"dry-rb/dry-types","tags":["types"],"tech":["utils"],"status":"model-ready"}
+  {"id":"rb.dry-types","imports":["dry-types"],"name":"dry-types","repo":"dry-rb/dry-types","tags":["types"],"tech":["utils"],"status":"model-ready"},
+  {"id":"rb.hanami","imports":[""],"name":"halnami","repo":"hanami/hanami","tags":["framework"],"tech":["ruby-web"],"status":"awaiting-model"}
```

Comment:
name: "hanami"

Reply:
Fixed the typo 👍 
I ran the "node verify.js" command and got the following result:
```
SUCCESS! technologies.json was updated; libs were sorted
```
The only changes introduce by running the command was the status from my addition to "model-ready" in the `libs/ruby.json` file, but I was not sure if I should keep this change since the instructions in the contributing guide requested to set this value to "awaiting-model".